kernel/src/simulationTools/LinearOSNS.hpp

```diff
@@ -266,6 +266,8 @@ class LinearOSNS : public OneStepNSProblem
     _keepLambdaAndYState = val ;
   }
 
+  virtual bool checkCompatibleNSLaw(NonSmoothLaw& nslaw) {return true;};
+  
```

Comment:
nitpick: why not a pure virtual so that a child class must implement this method?

Reply:
agree. Just I am lazy and I have not finished to implement all the classes. I will fix that at the end :-)

Reply:
Sounds good :)